docs/cli.md

```diff
@@ -2,84 +2,95 @@
 The project provides a command-line facilities for remotely 
 interacting with the topology subsystem.
 
-The commands are available at run-time using the consolidated `onos` client hosted in 
-the `onos-cli` repository, but their implementation is hosted and built here.
+The commands are available at run-time using the consolidated `onos` client hosted in the `onos-cli` repository.
 
-The documentation about building and deploying the consolidate `onos` client or its Docker container
+The documentation about building and deploying the consolidated `onos` client or its Docker container
 is available in the `onos-cli` GitHub repository.
 
 ## Usage
 ```bash
-> onos topo --help
-...
-```
+$ onos topo --help
+ONOS topology resource commands
 
-### Global Flags
-Since the `onos` command is a client, it requires the address of the server as well
-as the paths to the key and the certificate to establish secure connection to the 
-server.
+Usage:
+  onos topo [command]
 
-These options are global to all commands and can be persisted to avoid having to
-specify them for each command. For example, you can set the default server address
-as follows:
-```bash
-> onos topo config set address onos-topo-server:5150
-```
+Available Commands:
```

Comment:
You can refer to auto generated docs as well to avoid updating them everytime: 
https://github.com/onosproject/onos-cli/tree/master/docs/cli